lib/ssl/test/ssl_basic_SUITE.erl

```diff
@@ -986,3 +1012,42 @@ chipher_suite_checks(Version) ->
             ok
     end.
    
+many_client_starter(Hostname, Port, ClientOpts, Server) ->
+    spawn_clients(Hostname, Port, ClientOpts, 50),
+    ct:sleep(100),
+    many_client_starter(Hostname, Port, ClientOpts, Server).
+
+spawn_clients(_, _, _, 0) ->
+    ok;
+spawn_clients(Hostname, Port, ClientOpts, N) ->
+    spawn_link(fun() ->
+                     case N of
+                         20 ->
+                             exit(self(), kill);
+                         _ ->
+                             {ok, _} = ssl:connect(Hostname, Port, ClientOpts) 
+                     end
+               end),
+    spawn_clients(Hostname, Port, ClientOpts, N-1).
+
+loop(L) ->
+    {ok, A} = ssl:transport_accept(L),
+    spawn(fun() ->
+                  ssl:handshake(A)
+          end),
+    loop(L).
+                 
+check_process_count() ->
+    check_process_count(5).
```

Comment:
maybe minor detail but: I don't see value added for having check_process_count/0 function wrapper. I would probably skip adding it in that situation.

Reply:
It hides the implementation detail of waiting long enough for the processes that should have been terminated to terminate.
However, I add an argument so now it is a wrapper with one argument.